diagnostic.md

```diff
@@ -35,7 +37,12 @@ Record your responses inside the fenced code blocks below each question.
     ```
 
     ```md
-    <!-- your response here -->
+    1. The urls are the same but the file structure is different.  The first route
+       has product nested under products, thus the product folder can be found as a child
+       folder under the products folder.  The second route has products and product as siblings under the application folder.
+    2. When you want to display a 'product' in your browser, the first route will not
+       replace the content from 'products'.  Thus, every time a 'product' is viewed, the
+       content from 'products' will still be visible.  The second route will allow 'product' to replace content from 'products'.
```

Comment:
Looks Good.  Nested route definitions, do require a nested file structure.  This means that the Route and Template files for `'product'` are expected to be found at `app/products/product`.

One of the biggest distinctions is where Ember will render the content.  
The nested `'products'/'product'` instructs Ember to render
`app/products/product/template.hbs` in the `outlet` of the parent `app/products/template.hbs`.  
The non-nested `'product'` instructs Ember to render `app/product/template.hbs` in the context of the application template.